### DISCUSSION TITLE
Edit forum thread's tags with command

### DISCUSSION THREAD

**Discussion by civilCornball** [discord-php/DiscordPHP#1214] [Q&A] [answered]
How the heck do you edit an existing forum thread's tags via DiscordPHP?

Has to be somewhere in https://discord-php.github.io/DiscordPHP/classes/Discord-Parts-Thread-Thread, but all the ones I try don't seem to work.

($tagID is the tag's ID as a string, ex: 1231231231231231231)

I have tried:
`$thread->fill(['applied_tags' => $tagID]);`
`$thread->__set($tagID);`
`$thread->offsetSet('applied_tags', $tagID);`

And nothing changes.

It's possible that I may need to create my own JSON command - if so, I would need to research [how discord.py does it,](https://discordpy.readthedocs.io/en/latest/api.html?highlight=applied_tags#discord.Thread.edit) and modify the JSON command we use for $thread->rename()

**Comment by SQKo** [comment, marked as answer]
You are overcomplicating (please refrain from using the Part `__set()` or `offsetSet()` methods), to modify Part attributes, just set the property and use the corresponding repository's `save()` method as the basic explained https://github.com/discord-php/DiscordPHP/wiki#parts

Simply do `$thread->applied_tags = [1231231231231231231]`
And then use the repository save method `$threads->save($thread)->done()` where `$threads = $channel->threads`

**Reply by civilCornball** [reply]
I was unaware of `save()`, thank you!
